.github/workflows/release-dast.yml

```
@@ -244,41 +230,36 @@ jobs:
         include:
           - name: backend
             image_repo: app-stayhealthy-backend
-            image_digest: ${{ inputs.image_digest-backend }}
+            image_digest: ${{ inputs.image-digest-backenddigest-backend }}
```

actionlint (1.7.12)

[error] property "image-digest-backenddigest-backend" is not defined in object type {image-digest-backend: string; image-digest-frontend: string; image-digest-worker: string}

(expression)